src/default-listeners.coffee

```diff
@@ -69,8 +69,7 @@ module.exports = (robot, config) ->
 
   if activeModelNames.length isnt 0
 
-    # The robot hears ALL. You cannot run.
-    robot.catchAll (msg) ->
+    learning_listener = (msg) ->
```

Comment:
Can you rename this to `learningListener`? CoffeeScript convention (at least in this repo?) is for camelCase identifiers 🙇🏻